cli/main.go

```diff
@@ -43,10 +44,37 @@ func buildVersionString() string {
 			fmt.Fprintf(versionsWriter, "\t%s\n", image)
 		}
 	}
-	if refValues, err := json.MarshalIndent(manifest.EmbeddedReferenceValues(), "\t", "  "); err == nil {
-		fmt.Fprintf(versionsWriter, "embedded reference values:\t%s\n", refValues)
+
+	embeddedReferenceValues := manifest.EmbeddedReferenceValues()
+	for _, platform := range platforms.All() {
+		fmt.Fprintf(versionsWriter, "\nreference values for %s platform:\n", platform.String())
+
+		runtimeHandlerName, err := manifest.RuntimeHandler(platform)
+		if err != nil {
+			panic(fmt.Sprintf("Couldn't get runtime handler name for platform %s", platform.String()))
+		}
+		fmt.Fprintf(versionsWriter, "\truntime handler:\t%s\n", runtimeHandlerName)
+
+		switch platform {
+		case platforms.AKSCloudHypervisorSNP:
+			fmt.Fprintf(versionsWriter, "\tlaunch digest:\t%s\n", embeddedReferenceValues.AKS.TrustedMeasurement.String())
+			fmt.Fprint(versionsWriter, "\tdefault SNP TCB:\t\n")
+			fmt.Fprintf(versionsWriter, "\t    bootloader:\t%d\n", embeddedReferenceValues.AKS.SNP.MinimumTCB.BootloaderVersion.UInt8())
```

Comment:
```suggestion
			fmt.Fprintf(versionsWriter, "\t\tbootloader:\t%d\n", embeddedReferenceValues.AKS.SNP.MinimumTCB.BootloaderVersion.UInt8())
```
nit

Reply:
tabwriter does weird things if we use another tab:
![image](https://github.com/user-attachments/assets/8024af55-922f-413a-8c32-fb4e8ca78a36)

Reply:
Hm.. This seems very weird.

Reply:
Instead of handcrafting this output, can we maybe switch to a structured format that has opinionated rendering but decent readability, e.g. YAML or TOML?

Reply:
@katexochen ^

Reply:
Fine for me, if it contains the needed info.

Reply:
> Fine for me, if it contains the needed info.

The embedded reference values don't contain the genpolicy version.

Reply:
@burgerdev @katexochen are you fine with hand-crafting the ouput for now? Maybe once we only use a single genpolicy version, we can just print the reference values verbatim.